README.md

```diff
@@ -31,6 +31,8 @@ Labs teams must follow all [Labs Engineering Standards](https://labs.lambdaschoo
 
 ## Getting Started
 
+Here is a link to help set up your [local backend development environment](https://docs.google.com/document/d/109xl44p7A_f2oyT4PffSgINEuEd5ph6orSfaN4g-XBQ/edit?usp=sharing)
```

Comment:
Rather than a separate google doc, is there any reason we couldn't have these instructions listed out as a separate documentation markdown file somewhere in the back end repo? Something like how_to_deploy.md or similar in root?

Reply:
Hey Paul, I removed the Google doc link and replaced it with a new markdown file in the docs folder.